src/maps.py

```diff
@@ -10,12 +10,13 @@
 from itertools import chain
 from math import log
 from math import e
+import app as app
 
 chart_studio.tools.set_credentials_file(
-    username="chartstudiouser", api_key="m9KxT5JPEEukONNW8E50"
+    username="anirudhlakhotia", api_key="tcCFHV4CoCJaqmLfANU6"
```

Comment:
ok so for these api keys, what I did was monstrously stupid. Please try and change to using environment variables, using something like dotenv https://pypi.org/project/python-dotenv/

Reply:
Yep, fixed it